### ISSUE TITLE
Is there a account that can only be run by BP?

### ISSUE THREAD

**Issue by NeverLandFly** [EOSIO/eos#4796]
Assume the account is called 'system'.
If I create a account named 'auto' and set both owner and activer permissions to 'system',
then the action of 'auto' can only be executed by the current active BP.

**Comment by NeverLandFly** [comment]
@andriantolie Please correct me if i'm wrong. By my understanding, to make a contract run independently on an account( not affected by any individuals even myself), I should set a contract on this account and then change the permissions of the account to 'eosio'. Then it will run by the way the contract is coded without interference.

**Comment by andriantolie** [comment]
In that case, you will fully surrender that account ownership to 'eosio', which means only 2/3+1 producers can modify the contract, etc. If that's what you want, then it's the right way. But why would you surrender your account ownership to block producers?

**Comment by NeverLandFly** [comment]
@andriantolie  I thought this might solve the trust problem. I have considered the Dice example in the tutorial. 
1. If I'm the host (Dice) and I have the owner and active permission, then i'm able to transfer all of those margins from Alice and Bob without their permission. 

2. If I create a multisig Dice account and make the only possible way to transfer funds is that 3 parties(me, Alice, Bob) all agree. Assume Bob have lost most his money, then when Alice want to withdraw, he may simply ignore the request.

3.If I set the threshold to 2, Alice can still doubt that I mignt collude with Bob to steal her margin.

The only solution I came up with is that Dice run automatically. And when some event is triggered, say time limit is touched or both Alice and Bob withdrawed their funds, the contract will give the permissions back to me.

I think this will make everybody happy.

Am I thinking too much, and there is just an easy way to solve this?
